src/styles/blocks/main/about-us.scss

```
@@ -0,0 +1,59 @@
+.about-us {
+  background-color: #fcfaef;
+  &__bg-image {
+  height: 200px;
+  background-image: url(../images/main/about-us/about-us-bg-image.png);
+  background-position-y: 10%;
+  background-size: cover;
+  background-color: #fcfaef;
+
+  @include onTablet {
+    height: 400px;
+    background-position-y: 35%;
+  }
+
+  @include onDeskstop {
+    background-position-y: 65%;
+    height: 460px;
+  }
+  }
+ &__info {
```

Don't forget about good code formatting.
```suggestion
  background-color: #fcfaef;
  
  &__bg-image {
    height: 200px;
    background-image: url(../images/main/about-us/about-us-bg-image.png);
    background-position-y: 10%;
    background-size: cover;
    background-color: #fcfaef;
  
    @include onTablet {
      height: 400px;
      background-position-y: 35%;
    }
  
    @include onDeskstop {
      background-position-y: 65%;
      height: 460px;
    }
  }
  
 &__info {
```